apps/nestor-public/src/pages/view-dashboard.vue

```diff
@@ -288,12 +201,77 @@ async function getOrganisations() {
       return { ...row, hasSubmittedData: status };
     }
   );
-  OrganisationsChartData.value = data;
+  organisations.value = data;
+}
+
+async function getStats() {
+  const query = gql`
+    {
+      Components {
+        name
+        statistics {
+          id
+          label
+          value
+          valueOrder
+        }
+      }
+    }
+  `;
+  const response: IComponentsResponse = await request("../api/graphql", query);
+  const data: IComponent[] = response.Components;
+
+  const highlights = data.filter(
+    (row: IComponent) => row.name === "data-highlights"
+  );
+  registryHighlights.value = Object.fromEntries(
+    highlights[0]["statistics"].map((row: IStatistics) => [
+      row.label,
+      row.value,
+    ])
+  );
+
+  const sexData = data.filter(
+    (row: IComponent) => row.name === "pie-sex-at-birth"
+  );
+  sexAtBirth.value = Object.fromEntries(
+    sexData[0]["statistics"]
+      .map((row: IStatistics) => [row.label, row.value])
+      .sort((current: any[], next: any[]) => (current[1] < next[1] ? 1 : -1))
+  );
+
+  const age = data.filter((row: IComponent) => row.name === "barchart-age");
+  ageAtInclusion.value = age[0]["statistics"];
+
+  const maxValue: number = d3.max(
+    ageAtInclusion.value,
+    (row: IStatistics) => row.value
+  ) as number;
+  const ymax: number = Math.round(maxValue / 10) * 10;
+  ageAtInclusionTicks.value = seqAlongBy(0, ymax, 25);
```

Comment:
I think the `generateAxisTicks` function should be used here. If I add some values in the demo dataset, the y-axis ticks do not update. I think the step (`25`) is a bit too large.

In this [example in the CRANIO provider dashboard](https://github.com/molgenis/molgenis-emx2/blob/master/apps/cranio-provider/src/views/provider-overview.vue#L205-L216), it generates the y-axis ticks and makes some assumptions on the interval between each tick (e.g., by 5, 10, 15, etc.)